Sources/String+Sweet.swift

```diff
@@ -8,4 +8,11 @@ extension String {
 
         return rect.width
     }
+
+    func height(for font: UIFont, constrainedToWidth width: CGFloat) -> CGFloat {
+        let attributes = [NSFontAttributeName: font]
+        let rect = (self as NSString).boundingRect(with: CGSize(width: width, height: CGFloat.greatestFiniteMagnitude), options: .usesLineFragmentOrigin, attributes: attributes, context: nil)
```

Comment:
If this is to calculate height for arbitrary strings, it most definitely needs it, since as the API provider you have no way of knowing which one is being used for displaying. This should probably take the whole attributes dictionary, instead of just a font, since you want to calculate height using the same attributes as the label/text view.